### PR TITLE
First convection model

### PR DESCRIPTION
**Pull request by cometz95**

### CODE REVIEW

File: data/amars.yaml
Comment (chengcli) [inline data/amars.yaml:1]: You can remove this file since you already have it in examples

File: src/harp/set_spectral_properties.cpp [hunk 1]
```diff
@@ -46,6 +46,7 @@ void RadiationBand::SetSpectralProperties(AirColumn& ac, Real const* x1f,
       for (int m = 0; m < nspec; ++m) {
         auto& spec = pgrid_->spec[m];
         Real kcoeff = a->GetAttenuation(spec.wav1, spec.wav2, air);  // 1/m
+        // Real kcoeff = 5;
```
Comment (chengcli) [inline src/harp/set_spectral_properties.cpp]: remove this

**Review by chengcli — CHANGES_REQUESTED** [review]

**Review by chengcli — APPROVED** [review]

**Comment by cometz95** [comment]
merge amars convective model to canoe
-add input file for multi species condensation
-add vapor pressure files

**Comment by cometz95** [comment]
merge amars convective model to canoe
-add input file for multi species condensation
-add vapor pressure files

**Pull request closed without merging**